addons/applovin_max/Example/Scenes/main.gd

```diff
@@ -140,77 +186,118 @@ func _on_interstitial_ad_loaded(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo)
+	
 	
 func _on_interstitial_ad_displayed(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Interstitial ad displayed")
 	
+	
 func _on_interstitial_ad_display_failed(ad_unit_id: String, errorInfo: AppLovinMAX.ErrorInfo, ad_info: AppLovinMAX.AdInfo):
 	inter_button.disabled = false
 	inter_button.text = "Load Interstitial"
 	_log_message("Interstitial ad failed to display")
 	
+	
 func _on_interstitial_ad_clicked(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Interstitial ad clicked")
 	
+	
 func _on_interstitial_ad_hidden(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	inter_button.disabled = false
 	inter_button.text = "Load Interstitial"
 	_log_message("Interstitial ad hidden")
 
+
 ### Rewarded Ad Callbacks
 
 func _on_rewarded_ad_loaded(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	rewarded_button.disabled = false
 	rewarded_button.text = "Show Rewarded Ad"
 	_log_message("Rewarded ad loaded from" + ad_info.network_name)
 	
+	
 func _on_rewarded_ad_load_failed(ad_unit_id: String, errorInfo: AppLovinMAX.ErrorInfo):
-	_log_message("Rewarded ad failed to load with code " + str(errorInfo.code) + " with " + str(errorInfo.message))
+	_log_message("Rewarded ad failed to load with code " + str(errorInfo.code) + " with " + errorInfo.message)
+	
 	
 func _on_rewarded_ad_displayed(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Rewarded ad displayed")
 	
+	
 func _on_rewarded_ad_display_failed(ad_unit_id: String, errorInfo: AppLovinMAX.ErrorInfo, ad_info: AppLovinMAX.AdInfo):
 	rewarded_button.disabled = false
 	rewarded_button.text = "Load Rewarded Ad"
 	_log_message("Rewarded ad failed to display")
 	
+	
 func _on_rewarded_ad_clicked(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Rewarded ad clicked")
 
+
 func _on_rewarded_ad_received_reward(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo, reward: AppLovinMAX.Reward):
 	_log_message("Rewarded ad granted reward")
 	
+	
 func _on_rewarded_ad_hidden(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	rewarded_button.disabled = false
 	rewarded_button.text = "Load Rewarded Ad"
 	_log_message("Rewarded ad hidden")
 	
+	
 ### Banner Ad Callbacks
 
 func _on_banner_ad_loaded(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Banner ad loaded from" + ad_info.network_name)
 	
+	
 func _on_banner_ad_load_failed(ad_unit_id: String, errorInfo: AppLovinMAX.ErrorInfo):
 	_log_message("Banner ad failed to load with code " + str(errorInfo.code) + " with " + str(errorInfo.message))
 	
+	
 func _on_banner_ad_clicked(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Banner ad clicked")
 	
+	
 func _on_banner_ad_expanded(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Banner ad expanded")
 	
+	
 func _on_banner_ad_collapsed(ad_unit_id: String, ad_info: AppLovinMAX.AdInfo):
 	_log_message("Banner ad collapsed")
 	
+	
+### Banner Ad Callbacks
```

Comment:
Hmm so it actually doesn't look like you're re-using callbacks (per my coment above) - so you have some errors above to connect to the right callbacks